src/app/app.module.ts

```diff
@@ -45,6 +47,19 @@ import { MySpinnerComponent } from './extensions/my-spinner/my-spinner.component
         }),
         CoreModule.forRoot(),
         Ng2SmartTableModule,
+        SocialLoginModule,
+    ],
+    providers: [
+        {
+            provide: AuthServiceConfig,
+            useFactory: () => {
+                return new AuthServiceConfig([{
+                    id: GoogleLoginProvider.PROVIDER_ID,
+                    provider: new GoogleLoginProvider("1058475320740-nd03d7v9pd9m4np0fatrfadbdaoa3lf5.apps.googleusercontent.com")

```

Comment:
Please move it to environment variables: 1058475320740-nd03d7v9pd9m4np0fatrfadbdaoa3lf5.apps.googleusercontent.com

Reply:
and clear git history after